spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -734,12 +746,6 @@ def ensure_csrf(klass)
       # we don't have a valid csrf token, that's why we have to do at least one request
       block_given? ? yield : klass.all
 
-      # Update 18th August 2016
-      # For some reason, we have to query the resource twice to actually get a valid csrf_token
-      # I couldn't find out why, the first response does have a valid Set-Cookie header
-      # But it still needs this second request
```

Comment:
Would it make sense to write tests for the assumption that we can safely revert this old hack?